README.md

```diff
@@ -8,8 +8,8 @@ This microservice requires an LDAP server to run against.
 
 ### Prerequisites
 - Docker and Docker Compose installed (if you want to run the microservice inside Docker)
+- Python 3.12 installed on your machine (if you are not using Docker)
```

Comment:
As in https://github.com/ral-facilities/inventory-management-system-api/pull/202/files, should the requires-python in the pyproject.toml be updated as well if we are saying its a requirement in the readme?

Reply:
Ah yes, I completely forgot about this, apologies..